vcn_gateways.tf

```diff
@@ -44,6 +88,15 @@ data "oci_core_services" "all_oci_services" {
   count = var.service_gateway_enabled == true ? 1 : 0
 }
 
+data "oci_core_services" "object_storage" {
```

Comment:
What's the purpose of this? It doesn't seem it's used?

Reply:
The block at line 91 was used to give choice to user to create a service gateway route rule with either "All <region> Services" or "Object Storage in <region>".

Later, I chose to remove this feature and keep the existing fixed "all services" rule for the moment (line 146).

This data is currently not used, but will be needed when we transition to completely configurable rules (no rules created that the user cannot modify).

I can either let this block here in preparation for later transition or remove it: as you prefer.

Reply:
It seems redundant to me. Unless I'm missing something here

Reply:
I will remove it for now because not used in vcn module.
For route_table module, it will be useful to give choice whether we route all OSN traffic to SGW or only Object Storage, same option as in the console.

Reply:
Ok